CesiumGltfReader/src/decodeQuantized.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+namespace CesiumGltf {
+struct Model;
+}
+
+namespace CesiumGltfReader {
+
+void decodeQuantized(CesiumGltf::Model& model);
```

Comment:
It wouldn't hurt to be a little more descriptive here. `decodeQuantizedMeshExtension`, maybe.